scripts/raw_to_easymocap.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, shutil
+
+DATA_TYPE = ['png','PNG','jpg','JPG']
+def create_input():
+    USAGE = "USAGE: python path/to/raw_to_easymocap.py path/to/raw_images path/to/intri.yml path/to/extri.yml"
+    if(len(sys.argv) != 4) or '--help' in sys.argv or '-h' in sys.argv:
+        print(USAGE)
+        os._exit(0)
+    # Raw images path
+    path_img = sys.argv[1]
+    dataset_name = 'easymocap_input'
+    # Create output path and directory
+    output = os.path.join(dataset_name, 'videos')
+    os.makedirs(output)
+    # Check if files have correct extension
+    files = [i for i in os.listdir(path_img) if i.split('.')[-1] in DATA_TYPE]
```

Comment:
use `.endswith()`, better should be `Path.suffix` or even `Path.suffixes` to handle.

Reply:
Changed to `Path.suffix`